maven/prepare-release.sh

```diff
@@ -16,7 +16,7 @@ echo
 echo "Preparing release branch '${BRANCH}' ..."
 git branch ${BRANCH}
 git checkout ${BRANCH} && \
-(! git branch -r --contains ${BRANCH} &> /dev/null || \
+(! git branch -r | grep "${BRANCH}" &> /dev/null || \
```

Comment:
Sandwich the full remote name, otherwise e.g. `CELDEV-123` may be found for `CELDEV-1234`
```suggestion
(! git branch --remotes | egrep "(^|\s)origin/${BRANCH}(\s|$)" &> /dev/null || \
```